### ISSUE TITLE
This board will never power up the SOM

### ISSUE THREAD

**Issue by ajawamnet** [antmicro/google-coral-baseboard#2]
According to Schmoogle - you need to have pins 93 thru 100 on what you call JM3 (google SOM Dev board J1312) connected to 5 volt. 

But your board shows those pins connected to VSYS_5V via polyfuse F1 - and VSYS_5V goes nowhere - see the altium screenshot attached...

AND you show pin 93 tied to GND - not cool

See the screenshot of the Google SOM datasheet - where it states that you MUST connect those pins. 

What kind of man-bun, java-jockey crap is this?

![WTF-Over](https://user-images.githubusercontent.com/25396864/65918298-506ee800-e3a7-11e9-8e95-cb67f816944d.jpg)
![FromtheactualGoogleTPUSOMdevlayout](https://user-images.githubusercontent.com/25396864/65918299-506ee800-e3a7-11e9-85b6-9148407ff467.jpg)
![pagefromgoogleSOMdatasheet](https://user-images.githubusercontent.com/25396864/65918300-506ee800-e3a7-11e9-8298-e1df66aec9a0.jpg)

**Comment by pkatarzynski** [comment]
Hi
Thank you for your contribution.
We have updated power delivery for the SOM.

**Comment by ajawamnet** [comment]
OK - I'll look for the updated github files

-- 
WAM
http://www.ajawamnet.com
http://www.ajawamnet.com/ajawamnet/marketturd.htm
------------

More information on my lab and shop:
http://www.ajawamnet.com
http://www.ajawamnet.com/ajawam3/swarf/swarf.htm
------------------------
"Success is going from failure to failure without a loss of enthusiasm."
~~ Sir Winston Leonard Spencer-Churchill (1874-1965)

"I've failed at so many things so many times I make Abraham Lincoln look like Bill Gates"
~~ Wayne Mitzen (1959-  )

"What makes an expert isn't so much what they know,
It's that they've done similar things so many times wrong
They know what not to do"
~~ Wayne Mitzen (1959-  )

"You just keep pushing. You just keep pushing. I made every mistake that could be made. But I just kept pushing."
~~ Rene Descartes 1596-1650

"I hate the way people just go from ignorance to despair so quickly; it's the lazy man's option"
~~Professor Tim Flannery (1956-  )

"When you do things right, people won't be sure you've done anything at all."
~~ God, in Futurama episode "Godfellas"

"Two things are infinite: the universe and human stupidity;
and I'm not even sure about the universe."
~~ Albert Einstein (1879-1955)

"Some scientists claim that hydrogen, because it is so plentiful, is the basic building block of the universe.
I dispute that. I say there is more stupidity than hydrogen, and that is the basic building block of the universe."
~~ Frank Zappa (1940-1993)
